app/account/services_views.py

```diff
@@ -0,0 +1,24 @@
+import typing as tp
+
+
+def create_new_account(data: tp.Dict[str, tp.Union[str, bool, int]]) -> tp.Dict[str, tp.Union[str, bool, int]]:
+    """
+    Create new account
+    Args:
+        data: register data
+    Return:
+        dict with account data
+    """
+
+    from main import db
+    from models import Account
+
+    data['is_staff'] = False
+
+    new_account = Account(**data)
```

Comment:
it's not good practice to modify incoming data, rather do
```suggestion
    new_account = Account(**data, is_stuff=False)
```

Reply:
done